config/runtime/kunit.jinja2

```diff
@@ -152,5 +152,4 @@ cd {src_path}
         api_helper = kernelci.api.helper.APIHelper(self._api)
         api_helper.submit_results(result, self._node)
 
-        return node
```

Comment:
It looks like this bit should have been adjusted to `return self._node` in https://github.com/kernelci/kernelci-pipeline/commit/33fa931f20a1fceb2e278989e4c4c13da75e02d3.

I'd suggest fixing it here to comply with superclass function signature (i.e. returning updated Node) but I'm not 100% sure if this case applies here :thinking: (it made sense back in [`42c2ceb`](https://github.com/kernelci/kernelci-pipeline/commit/42c2cebe45d4abdf0f3be8981b7b722a0ad239fd#diff-4e48889b22e74380181ae303dd5372a80219e386747e5830863a06a26ee6519cL84))

Reply:
Thanks for digging in to find relevant commits. Your suggestion makes sense but the returned value is not being used anywhere.
The `main` function, once the job is generated would look like this:
```
def main(args):
    # Sometimes container DNS is failing if k8s node is still booting
    # This is a workaround:
    # test www.google.com until it works or retry 10 times
    for _ in range(30):
        try:
            socket.gethostbyname('www.google.com')
            break
        except socket.gaierror:
            pass
        # sleep 1 second
        time.sleep(1)
    else:
        print("DNS resolution failed, exiting", file=sys.stderr)
        sys.exit(1)

    job = Job(api_config=API_CONFIG_YAML, node=NODE, workspace=WORKSPACE)
    try:
        results = job.run()
        job.submit(results)
    except requests.exceptions.HTTPError as ex:
        print(ex, file=sys.stderr)
        detail = ex.response.json().get('detail')
        if detail:
            print(detail, file=sys.stderr)
        results = None
    except Exception:
        print(traceback.format_exc())
        results = None

    return results


if __name__ == '__main__':
    results = main(sys.argv)
    sys.exit(1 if results is None else 0)
```
`job.submit` internall calls `job._submit` and its returned value is not being used.

Reply:
What do you think about the above comment @pawiecz ?

Reply:
I agree that it makes no sense to forcefully keep the return value if it's not used anymore.

I'd suggest checking if the superclass method is called the same way (discarding returned Node) and align them so that such a mismatch doesn't bite us in the long run (e.g. during debugging)

Reply:
Thanks for the suggestion. Created https://github.com/kernelci/kernelci-core/pull/2582